internal/discovery/collector/publishing_test.go

```diff
@@ -4,20 +4,23 @@ package collector_test
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"io"
 	"net/http"
 	"os"
 	"testing"
 
 	"github.com/stretchr/testify/suite"
+	"github.com/trento-project/agent/internal/core/cloud"
 	"github.com/trento-project/agent/internal/discovery/collector"
 	"github.com/trento-project/agent/internal/discovery/mocks"
 	"github.com/trento-project/agent/test/helpers"
 )
 
 const (
-	apiKey        = "some-api-key"
-	discoveryType = "sap_system_discovery"
+	apiKey             = "some-api-key"
+	sapSystemdiscovery = "sap_system_discovery"
```

Comment:
nitpick: most probably we should remove this `global` const and have the discovery type in each test. What do you think?

Reply:
I don't mind either way. There's just some extra repetition but I'm fine with it.
Here's the commit a5bddc2b5eeb66d80faa8375c32f73406cc410e1

Reply:
Lol there was a reason, ci lint fails.
Fixed here 8273b25694c04d26122a9c43e33e8764cf308544